test/cmd/basicresources.sh

```diff
@@ -56,7 +56,7 @@ echo "routes: ok"
 
 # Expose service as a route
 oc create -f test/integration/fixtures/test-service.json
-[ ! "$(oc expose service frontend --create-external-load-balancer)" ]
+[ ! "$(oc expose service frontend --type="LoadBalancer")" ]
```

Comment:
`--type` is string (https://github.com/kubernetes/kubernetes/blob/6f4bcd5fe2fb4ebfda52b4169816e057ea73ba80/pkg/kubectl/cmd/expose.go#L76)